src/app/about/page.tsx

```diff
@@ -1,19 +1,49 @@
+import React from "react";
 import Image from "next/image";
 import styles from "./About.module.css";
 
 import OfficerCard from "./officer-card";
-import officerCardData from "./officer-data";
+import { positionData } from "./officer-data";
 
 export default function About() {
-  const officerCards = officerCardData.map((cardData) => (
-    <OfficerCard
-      key={cardData.position}
-      position={cardData.position}
-      image={cardData.image}
-      name={cardData.name}
-      socialLinksData={cardData.socialLinksData}
-    />
-  ));
+  // const officerCards = officerCardData.map((cardData) => (
+  //   <OfficerCard
+  //     key={cardData.position}
+  //     position={cardData.position}
+  //     image={cardData.image}
+  //     name={cardData.name}
+  //     socialLinksData={cardData.socialLinksData}
+  //   />
+  // ));
+
+  const officerCardSection = Object.entries(positionData).map(
+    ([semester, officers]) => {
+      const officerCards = Object.entries(officers).map(
+        ([position, officer]) => {
+          if (officer === undefined) {
+            return <div key={`${semester} ${position}`} />;
+          }
+
+          return (
+            <OfficerCard
+              key={`${semester} ${position}`}
+              position={position}
+              image={officer.image}
+              name={officer.name}
+              socialLinksData={officer.socialLinksData}
+            />
+          );
+        },
+      );
+
+      return (
+        <React.Fragment key={semester}>
```

Comment:
[Comment]: Great use of `React.Fragment` here - definitely how I would have gone about this to avoid some unnecessary div wrapping.